### PR TITLE
bring back popupcoords under admin control

### PR DESCRIPTION
**Pull request by ReuschelCGN**
- fix Lint
- update german translations
- make userslider for coords in popup shareable in config (for gyms/stops/mons/portals)
  (default: false)
- fix Lint another time (i hate that)

### CODE REVIEW

File: server/src/services/checkForUpdates.js
Comment (clburlison) [inline server/src/services/checkForUpdates.js:46]: This shouldn't be here 
```suggestion
            `git ls-remote https://github.com/WatWowMap/ReactMap/ refs/heads/${branch}`,
```

File: server/src/services/ui/clientOptions.js
Comment (TurtIeSocks) [inline server/src/services/ui/clientOptions.js:7]: One small thing here, instead of importing them like this, please consolidate it to be like this:
```js
const {
  clientSideOptions,
  map: { enableMapJsFilter, enablePokemonPopupCoordsSelector, ...the rest },
  api: {
    pvp: { levels },
  },
} = require('../config')
```
replacing `the rest` with the other imports.

**Review by TurtIeSocks — CHANGES_REQUESTED** [review]
Thanks for the quick turn around. Apologies, I should have checked with you before merging in Pliny's PR. Left a comment on one small thing. You mentioned you didn't lint but the linting commands didn't make any changes so I guess it's good to go? Either way, in the future, if you weren't aware, you can run `yarn prettier:fix && yarn eslint:fix` to save some time.

**Review by TurtIeSocks — APPROVED** [review]

**Pull request merged**
